tools/setup_buildenv_ubuntu20.04.sh

```diff
@@ -118,11 +134,19 @@ if [[ $PAUSEAFTEREACHLINE == "true" ]]; then
   read
 fi
 
-echo "=== Step 14: Removing the downloaded archive and build folder of USB DFU host utility ==="
+echo "=== Step 15: Removing the downloaded archive and build folder of USB DFU host utility ==="
 rm dfu-util-0.10.tar.gz
 rm -rf dfu-util-0.10
 if [[ $PAUSEAFTEREACHLINE == "true" ]]; then
   echo "Step finished."
 fi
 
-echo "Finished setting up EdgeTX v2.4 development environment. Please reboot the computer."
+if [[ $I386ARCH == "no" ]]; then
+## Likely running in WSL2
+  echo "Reloading .bashrc".
+  source ~/.bashrc
```

Comment:
As I mentioned before, this has never, and will never work - on bare metal or WSL2, for two reasons. `source`, when run in a shell script, results in that script being imported into the shell scripts session, not the user session. So it only lasts until the script exits (think local variable instead of global variable). And secondly, this does not work on Ubuntu because the first thing the .bashrc checks for is "am I running in an interactive session - if no, goodbye", so it exits immediately.

This should instead be a message to the user to run that command upon the script completing. 

This also leads to another question... for what reason does the user actually need to reboot their computer after running this script? With the exception of the `apt update` I'm not seeing anything that requires it, and that doesn't require reboots often either. At most, the same "Run `source ~/.bashrc` to load the updated path settings" type instruction should be sufficient there also.

Reply:
Thanks for the feedback. I removed the command in script and replaced it with a hint for user. Also removed the reboot recommendation.